packages/playwright/docs/index.mdx

```diff
@@ -112,6 +114,37 @@ For tailored visual testing, the `data-visual-test` attributes provide control o
 <div id="clock" data-visual-test="transparent">...</div>
 ```
 
+## Create multiple Argos builds from a single suite of tests
+
+You may want to create multiple Argos builds from a single suite of Playwright tests. To achieve this, you can use a dynamic `buildName`. Each different build name creates a new build that contains only a specific set of screenshots.
+
+```ts title="playwright.config.ts"
+import { defineConfig } from "@playwright/test";
+import { createArgosReporterOptions } from "@argos-ci/playwright/reporter";
+
+export default defineConfig({
+  // ... other configurations
+
+  reporter: [
+    // Use "dot" reporter on CI, "list" otherwise (Playwright default).
+    process.env.CI ? ["dot"] : ["list"],
+
+    // Argos reporter
+    [
+      "@argos-ci/playwright/reporter",
+      createArgosReporterOptions({
+        uploadToArgos: !!process.env.CI,
+        buildName: {
+          values: ["app", "website"],
+          // Split Argos build based on Playwright tags
```

Comment:
// Split Argos build based on Playwright's test name (test tag)

Reply:
It's not the same thing. See https://playwright.dev/docs/test-annotations#tag-tests